elasticdl/python/ps/servicer.py

```diff
@@ -249,3 +244,12 @@ def _save_params_to_checkpoint_if_needed(self):
                 shard_index=self._ps_id,
                 shard_num=self._num_ps_pods,
             )
+
+    def _set_optimizer_learning_rate(self, learning_rate):
+        if learning_rate is None:
```

Comment:
What is the default value for a float in proto? 0?

Reply:
For numeric types, the default value is zero and 0.0 for a float. https://developers.google.com/protocol-buffers/docs/proto3#default
And I will fix it using `if learning_rate == 0.0:`